### DISCUSSION TITLE
使用自定义顶栏的搜索栏时会出现模块组件的选项，如何去除？

### DISCUSSION THREAD

**Discussion by madokami1** [the1812/Bilibili-Evolved#4442] [提问 / Q&A]
使用自定义顶栏的搜索栏时会出现模块组件的选项，配置是以前v1导入的
![image](https://github.com/the1812/Bilibili-Evolved/assets/62499230/32486477-1c25-44c1-be68-4f111f1a8389)
![image](https://github.com/the1812/Bilibili-Evolved/assets/62499230/25af87e8-6126-44c0-901f-b6d00d5d5b26)
我想要沉浸式体验啊。。不用的时候怎么让他不跳出来

**Comment by Geeyun-JY3** [comment]
目前应该是没有相关设置项的（至少无 GUI）

**Reply by madokami1** [reply]
我之前用的是7月底的最新版本插件，导入同样V1设置的搜索栏还没有这个情况，安装当时的旧版本有用吗？此外试了试从旧电脑的篡改猴上到处1.12.2的插件，导入到新电脑上用不了，插件启动了但是界面没变，这个是为什么？旧电脑上的还生效，浏览器和篡改候版本一致。

**Reply by madokami1** [reply]
就是想弄成我以前这样的搜索栏。我有旧版本插件的油猴导出数据，也有旧版本插件内部导出的V1数据，可怎么样都恢复不了
![QQ图片20230918130450](https://github.com/the1812/Bilibili-Evolved/assets/62499230/e8742d5b-349d-4815-ae58-eddd663bb7a9)

**Reply by madokami1** [reply]
2.7.6和2.7.4导入V1是最上面那种搜索栏，导入油猴备份的1.12又没反应

**Reply by madokami1** [reply]
看到了不解决旧版本问题。。。我可以付费，大佬方便帮我看看吗？

**Reply by Geeyun-JY3** [reply]
> 我之前用的是7月底的最新版本插件，导入同样V1设置的搜索栏还没有这个情况，安装当时的旧版本有用吗？

据我所知 v2 以来搜索栏就是这样了，不只是搜索还能执行命令，你导入 v1 的设置也是这样的，装 v2 旧版最多是少些后续新版本实装的命令：https://github.com/the1812/Bilibili-Evolved/discussions/2195#discussioncomment-1379985

> 此外试了试从旧电脑的篡改猴上到处1.12.2的插件，导入到新电脑上用不了，插件启动了但是界面没变，这个是为什么？

v1 到 v2 一直都有人反馈界面出不来，原因乱七八糟，也可能有后续新版本修的 bug，我只能建议你检查网络环境；而且旧版本也不维护了，新电脑实在用不了也没办法。

**Comment by madokami1** [comment]
> > 我之前用的是7月底的最新版本插件，导入同样V1设置的搜索栏还没有这个情况，安装当时的旧版本有用吗？
> 
> 据我所知 v2 以来搜索栏就是这样了，不只是搜索还能执行命令，你导入 v1 的设置也是这样的，装 v2 旧版最多是少些后续新版本实装的命令：[#2195 (reply in thread)](https://github.com/the1812/Bilibili-Evolved/discussions/2195#discussioncomment-1379985)
> 
> > 此外试了试从旧电脑的篡改猴上到处1.12.2的插件，导入到新电脑上用不了，插件启动了但是界面没变，这个是为什么？
> 
> v1 到 v2 一直都有人反馈界面出不来，原因乱七八糟，也可能有后续新版本修的 bug，我只能建议你检查网络环境；而且旧版本也不维护了，新电脑实在用不了也没办法。

那请问现在还有v1的下载链接吗？翻了下没找到。请问网络环境具体是指什么？v1也有联网需求的吗

**Reply by Geeyun-JY3** [reply]
仓库里还放着 v1 最后一个离线版啊：https://github.com/the1812/Bilibili-Evolved/blob/master/bilibili-evolved.offline.user.js
我不好说 v1 有没有什么图标在你的地区需要翻墙才加载的出来，这问题印象中 v1 到 v2 都一直有人在反馈，所以建议检查网络环境，最好先翻一次看看能不能加载出来。
还有刚想起一个就是建议 v1 v2 脚本不要同时开启，可能会有问题。

***

另外为啥要死守 v1 呢？我觉得还不如让作者看看能不能给加个强迫症选项。

***

还有就 https://github.com/the1812/Bilibili-Evolved/discussions/4442#discussion-5639902 的图 2 来看，命令相关的搜索匹配得优化一下@the1812

**Comment by madokami1** [comment]
> 仓库里还放着 v1 最后一个离线版啊：https://github.com/the1812/Bilibili-Evolved/blob/master/bilibili-evolved.offline.user.js 我不好说 v1 有没有什么图标在你的地区需要翻墙才加载的出来，这问题印象中 v1 到 v2 都一直有人在反馈，所以建议检查网络环境，最好先翻一次看看能不能加载出来。 还有刚想起一个就是建议 v1 v2 脚本不要同时开启，可能会有问题。
> 
> 另外为啥要死守 v1 呢？我觉得还不如让作者看看能不能给加个强迫症选项。
> 
> 还有就 [#4442 (comment)](https://github.com/the1812/Bilibili-Evolved/discussions/4442#discussion-5639902) 的图 2 来看，命令相关的搜索匹配得优化一下@the1812

喔喔喔这个可以。感谢大佬。死守V1只能是习惯吧...看惯的UI不想改了，主要就是那个搜索栏跳出命令看着很别扭。网络问题...我的梯子是开机自启的，可能有什么地方不对吧。不过这个离线版可以用。感谢！
